### PR TITLE
Feat: Implement the functions in the ValidateAccount library

### LINKED ISSUE #247: Feat: Implement the functions in the `ValidateAccount` library.
## Description

Implement the internal `validate_account` function in the `AccountUtils` library.

File: `src/utils/validate_account.cairo`.

Function signature:
```rust
    /// Validates an account.
    /// # Arguments
    /// `account` - Account to validate.
    fn validate_account(account: ContractAddress);
```
---
Implement the internal `validate_receiver` function in the `ValidateAccount` library.

File: `src/utils/validate_account.cairo`.

Function signature:
```rust
    /// Validates a receiver.
    /// # Arguments
    /// * `receiver` - Account to validate.
    fn validate_receiver(receiver: ContractAddress);
```

## Acceptance criteria

- Function is implemented and well commented.
- Associated unit tests (nominal and failure) are implemented and passing.

## Resources

- [validateAccount function in GMX repo](https://github.com/gmx-io/gmx-synthetics/blob/77a2ff39f1414a105e8589d622bdb09ac3dd97d8/contracts/utils/AccountUtils.sol#L8)
- [validateReceiver function in GMX repo](https://github.com/gmx-io/gmx-synthetics/blob/77a2ff39f1414a105e8589d622bdb09ac3dd97d8/contracts/utils/AccountUtils.sol#L14)

### PR DESCRIPTION
**Pull request by delaaxe**
Closes #247

### CODE REVIEW

File: src/utils/validate_account.cairo
Comment (delaaxe) [inline src/utils/validate_account.cairo:1]: this file would be named `account_utils.cairo` if we use same as in GMX repo

File: src/utils/validate_account.cairo [hunk 1]
```diff
@@ -7,11 +7,13 @@ use starknet::ContractAddress;
 /// Validates an account.
 /// # Arguments
 /// * `account` - Account to validate.
-fn validate_account(account: ContractAddress) { // TODO
+fn validate_account(account: ContractAddress) {
+    assert(account.is_non_zero(), 'satoru/null-account');
```
Comment (zarboq) [inline src/utils/validate_account.cairo:11]: error message in a constant in an error file, following conventions of other error messages

File: src/utils/validate_account.cairo [hunk 2]
```diff
@@ -7,11 +7,13 @@ use starknet::ContractAddress;
...
 }
 
 /// Validates a receiver.
 /// # Arguments
 /// * `receiver` - Account to validate.
 fn validate_receiver(receiver: ContractAddress) { // TODO
+    assert(receiver.is_non_zero(), 'satoru/null-receiver');
```
Comment (zarboq) [inline src/utils/validate_account.cairo:18]: same

**Review by zarboq — CHANGES_REQUESTED** [review]
Left some comments for errors. You can create an error.cairo file in utils for that

File: src/utils/validate_account.cairo (hunk 1 above)
Comment (delaaxe) [reply src/utils/validate_account.cairo]: thanks didn't catch that

File: src/utils/validate_account.cairo (hunk 2 above)
Comment (delaaxe) [reply src/utils/validate_account.cairo]: done

File: tests/utils/test_validate_account.cairo [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+use satoru::utils::validate_account::{validate_account, validate_receiver};
+
+#[test]
+fn test_validate_account() {
+    let account = contract_address_const::<0x69420>();
+    validate_account(account);
+}
+
+#[test]
+#[should_panic(expected: ('satoru/null-account',))]
```
Comment (zarboq) [inline tests/utils/test_validate_account.cairo:11]:
```suggestion
#[should_panic(expected: ('null_account',))]
```

File: tests/utils/test_validate_account.cairo [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+#[test]
+fn test_validate_receiver() {
+    let account = contract_address_const::<0x69420>();
+    validate_receiver(account);
+}
+
+#[test]
+#[should_panic(expected: ('satoru/null-receiver',))]
```
Comment (zarboq) [inline tests/utils/test_validate_account.cairo:24]:
```suggestion
#[should_panic(expected: ('null_receiver',))]
```

File: src/utils/validate_account.cairo
Comment (zarboq) [reply src/utils/validate_account.cairo:1]: Good catch ! You can modify the name of the file also

**Review by zarboq — CHANGES_REQUESTED** [review]
Small modifications in tests then it should be good

**Review by zarboq — APPROVED** [review]

**Pull request merged**